docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -223,9 +237,16 @@ public StartContainerCmd createHostConfig(StartContainerCmd hostConfig) {
         if( dnsHosts.length > 0 )
             hostConfig.withDns(dnsHosts);
 
-        // ?
-        //if (volumes.length > 0)
-        //    hostConfig.with VolumesFrom (volumes);
+        if(volumesString!=null) {
+            for (String volume : splitAndFilterEmpty(volumesString)) {
+                try {
+                    hostConfig.withBinds(Bind.parse(volume));
+                } catch (IllegalArgumentException e) {
+                    LOGGER.log(Level.WARNING,"Cannot create volume "+volume+" when starting host");
+                    //hostConfig.withVolumes(Volume.parse(volume));
```

Comment:
You can't create volume during start.
Create container support bind + volume creation calls.
Start only support additional binding. 
In this plugin case it should be enough to configure container during creation.